teachertool/src/utils/index.ts

```diff
@@ -11,4 +11,21 @@ export function makeNotification(
         duration,
         expiration: Date.now() + duration,
     };
+}
+
+export const isLocal = () => {
```

Comment:
`isLocal` -> fyi we have this method already in pxt.util.

Reply:
Ah, I forgot that I can just call it that way. I think I found it there and just copied it over, but I'll change it to be the function call in pxt.util instead. Thanks!